### ISSUE TITLE
On Windows, python 2.7 conflicts with flang?

### ISSUE THREAD

**Issue by roryyorke** [python-control/Slycot#48]
I'm trying to setup CI builds on Appveyor, see [here](https://ci.appveyor.com/project/roryyorke/slycot/builds/22076616).  The Python 3 (3.5, 3.6, and 3.7) builds seem to be fine.

It looks like Python 2.7 and conda-forge's flang aren't compatible.  Is that right?

I don't know if that's a  problem, but perhaps we should note it somewhere.

The error is:

```
conda.exceptions.UnsatisfiableError: The following specifications were found to be in conflict:
  - flang
  - python=2.7
```

**Comment by repagh** [comment]
Yes. 
I don't think it is very productive to try to fix this issue with python 2.7, deprecation is around the corner.
People insisting on using python 2.7, windows and slycot in combination may get the slycot from cgohlke's page

**Comment by roryyorke** [comment]
Incompatibility noted in #52.
